config/default.yaml

```diff
@@ -1,4 +1,5 @@
 listen:
+  hostname: ''
```

Comment:
By default bind on  `127.0.0.1`

Reply:
Alright, I did it for the production example, but I didn't want to break the default config.